### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by EthanVanderHorn**
some bug fixes for things that were created by conformer support

### CODE REVIEW

File: nanome/_internal/_structure/_helpers/_conformer_helper.py [hunk 1]
```diff
@@ -139,6 +138,7 @@ def convert_to_conformers(complex, force_conformer = None): #Data.Complex -> Dat
                     off = 0
                     if name_hash in names_dictionary:
                         off = names_dictionary[name_hash]
+                    off +=1
```
Comment (EthanVanderHorn) [inline nanome/_internal/_structure/_helpers/_conformer_helper.py]: This was a huge bug with the to_conformer code that basically caused all atoms in a residue of the same element to become 1 atom in the conformer.

File: nanome/_internal/_structure/_io/_pdb/structure.py [hunk 2]
```diff
@@ -15,9 +15,9 @@ def structure(content):
     for i in range(num_molecules):
         molecule = structure_molecule(atoms_by_molecule[i], content.compnds)
         molecule._name = str(i)
-        complex._molecules.append(molecule)
+        complex._add_molecule(molecule)
     # Done
-    return complex._convert_to_frames()
+    return complex._convert_to_conformers()
```
Comment (EthanVanderHorn) [inline nanome/_internal/_structure/_io/_pdb/structure.py]: This should have been to_conformer. This was hiding the bigger bug.

File: testing/utilities.py [hunk 3]
```diff
@@ -354,4 +354,87 @@ def bond_atoms(atom1, atom2):
     bond.atom1 = atom1
     bond.atom2 = atom2
     atom1.residue._add_bond(bond)
-    return alter_object(bond)
\ No newline at end of file
+    return alter_object(bond)
```
Comment (EthanVanderHorn) [inline testing/utilities.py]: Useful timer for benchmarking.

File: testing/utilities.py [hunk 4]
```diff
@@ -354,4 +354,87 @@ def bond_atoms(atom1, atom2):
...
+
+class DebugTimer():
+    def __init__(self):
+        self.nano = 10**9
+        self._open_process = {}
+        self._closed_process = {}
+
+    def _get_time(self):
+        return time.clock()
```
Comment (Ramjii) [inline testing/utilities.py]: Huh? What happened here?

**Pull request merged**